src/test/java/seedu/guest/storage/StorageManagerTest.java

```diff
@@ -54,15 +54,15 @@ public void addressBookReadSave() throws Exception {
          * {@link JsonAddressBookStorage} class.
          * More extensive testing of UserPref saving/reading is done in {@link JsonAddressBookStorageTest} class.
          */
-        GuestBook original = getTypicalAddressBook();
-        storageManager.saveGuestBook(original);
-        ReadOnlyGuestBook retrieved = storageManager.readGuestBook().get();
+        GuestBook original = getTypicalGuestBook();
+        hotelStorageManager.saveGuestBook(original);
+        ReadOnlyGuestBook retrieved = hotelStorageManager.readGuestBook().get();
         assertEquals(original, new GuestBook(retrieved));
     }
 
     @Test
     public void getAddressBookFilePath() {
```

Comment:
Should the method name be getGuestBookFilePath() instead of getAddressBookFilePath()?